PopPUNK/__main__.py

```diff
@@ -470,16 +504,19 @@ def main():
             genomeNetwork = indivNetworks[min(rank_list)]
 
         # Ensure all in dists are in final network
-        networkMissing = set(map(str,set(range(len(refList))).difference(list(genomeNetwork.vertices()))))
+        vertex_list = set(get_vertex_list(genomeNetwork, use_gpu = args.gpu_graph))
+        networkMissing = set(set(range(len(refList))).difference(vertex_list))
         if len(networkMissing) > 0:
             missing_isolates = [refList[m] for m in networkMissing]
             sys.stderr.write("WARNING: Samples " + ", ".join(missing_isolates) + " are missing from the final network\n")
+            sys.stderr.write("These correspond to indices " + ", ".join(map(str,networkMissing)) + "\n")
```

Comment:
Is this sometimes being hit with the GPU graph?

Reply:
It should probably be an error rather than a warning

Reply:
Changed to error in 845acb0.